api/v1alpha1/ratelimitfilter_types.go

```diff
@@ -99,6 +99,11 @@ type RateLimitSelectCondition struct {
 	// +optional
 	// +kubebuilder:validation:MaxItems=16
 	Headers []HeaderMatch `json:"headers,omitempty"`
+
+	// SourceIPCIDR defines the match attributes for masked source IP addresses from `X-Forwarded-For` header.
+	// For example, `192.168.0.1/32`, `192.168.0.0/24`, `001:db8::/64`.
+	// +optional
+	SourceIPCIDR *string `json:"sourceIPCIDR,omitempty"`
```

Comment:
sorry for the churn here, but I think the field name `SourceIP` is more apt, and the doc string can highlight `subnet` or `cidr`.

Reply:
done